tf_publisher:
  ros__parameters:
    l: [1.0, 1.0, 1.0, 1.0, 1.0]
    alignments:
      joint_0: [0, 0, 0]
      joint_1: [1.5707963267948966, 0, 0]
      joint_2: [0, 0, 0]
      joint_3: [0, 0, 0]
      